Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -758,14 +758,20 @@ script:
       - 'true'
       - 'false'
   - arguments:
-    - description: The status of the endpoint to filter.
+    - description: A comma-separated list of endpoint statuses to filter.
       name: status
       auto: PREDEFINED
       predefined:
```

Comment:
Just noticed this,
If we want to allow a comma separated list as a command argument input, we should add `isArray: true`.
Also, for the same `status` argument, we have a predefined list of values, which as far as i know allows only one value as an input and not an array.
I think it should be one or the other and not both.
If the intended use is as your screenshot in the PR's description, i think the array option is the way to go here.
(The predefined list of options can be added in the description to help the users).